action/protocol/account/protocol_test.go

```diff
@@ -61,14 +61,14 @@ func TestLoadOrCreateAccountState(t *testing.T) {
 	addrv1 := identityset.Address(27)
 	s, err := accountutil.LoadAccount(sm, addrv1)
 	require.NoError(err)
-	require.Equal(s.Balance, state.EmptyAccount().Balance)
+	require.Equal(s.Balance, big.NewInt(0))
 
 	// create account
 	require.NoError(createAccount(sm, addrv1.String(), big.NewInt(5)))
 	s, err = accountutil.LoadAccount(sm, addrv1)
 	require.NoError(err)
 	require.Equal("5", s.Balance.String())
-	require.Equal(uint64(0x0), s.Nonce)
```

Comment:
add test to create new type account, assert `s.PendingNonce() == 0`

Reply:
Are you sure to add unit test to this PR?